src/vishakha/FindPosNegAndZero.java

```diff
@@ -0,0 +1,49 @@
+/*Create a class where user will provide int array. *[ideal Expected Time : 15 mins, Max 
+    Time : 25 mins]*
+
+    A) User can provide any positive and negative number
+    B) You need to find out how many positive, negative numbers and zeros are there in array */
+
+package vishakha;
+
+import java.util.Scanner;
+
+public class FindPosNegAndZero {
+	int posCount;
+	int negCount;
+	int zeroCount;
+	
+	void findNoType(int[] num){
+		for(int index=0; index<num.length; index++){
+			if(num[index]>0)
+				posCount++;
+			else if(num[index]<0)
+				negCount++;
+			else
+				zeroCount++;
+		}
+		displayNoType();
+	}
+	
+	void displayNoType(){
+		 if(posCount>0)
+			 System.out.println("There are "+posCount+ " positive numbers in the array");
```

Comment:
Here I don't think so we require if condition here as, this is just for display purpose method